automation/check-npm-version.js

```diff
@@ -27,24 +29,6 @@ function semverGte(v1, v2) {
 	return true;
 }
 
-function _testSemverGet() {
```

Comment:
Typescript wasn't happy about this unused function, the `_` prefix only has special meaning in actual typescript code